src/index.js

```diff
@@ -137,6 +137,17 @@ async function main() {
 			await setBalance(relayChainApi, account, balance)
 		}
 	}
+
+	// wait here until the next session. Currently it's configured to be 5 blocks
+	// with 6 seconds per block. We wait for 2 sessions just in case.
+	let secsToSleep = 2 * 5 * 6
```

Comment:
Hard coding this is not a good idea :see_no_evil: 

There exists a constant `EpochDuration` in babe that you can read to get the number of blocks per session.

Reply:
Alternatively, we can configure it ourselves. Ideally, though, we just wait for the right condition, specifically, that the both parachains are registered.

Reply:
Done just that in the d9b6708e5175244aa5c83685aa5d0633b837a1fe